bundles/binding/org.openhab.binding.zwave/database/philio/pat02c.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+    <Model>PAT02-C</Model>
+    <Label lang="en">Flood Sensor</Label>
+    <CommandClasses>
+        <Class><id>0x20</id></Class>    <!-- COMMAND_CLASS_BASIC -->
+        <Class><id>0x30</id></Class>    <!-- COMMAND_CLASS_SENSOR_BINARY_V2 -->
+        <Class><id>0x31</id></Class>    <!-- COMMAND_CLASS_SENSOR_MULTILEVEL_V5 -->
+        <Class><id>0x59</id></Class>    <!-- COMMAND_CLASS_ASSOCIATION_GRP_INFO -->
+        <Class><id>0x5A</id></Class>    <!-- COMMAND_CLASS_DEVICE_RESET_LOCALLY -->
+        <Class><id>0x5E</id></Class>    <!-- COMMAND_CLASS_ZWAVEPLUS_INFO_V2 -->
+        <Class><id>0x70</id></Class>    <!-- COMMAND_CLASS_CONFIGURATION -->
+        <Class><id>0x71</id></Class>    <!-- COMMAND_CLASS_NOTIFICATION_V4 -->
+        <Class><id>0x72</id></Class>    <!-- COMMAND_CLASS_MANUFACTURER_SPECIFIC_V2 --> 
+        <Class><id>0x73</id></Class>    <!-- COMMAND_CLASS_POWERLEVEL -->
+        <Class><id>0x7A</id></Class>    <!-- COMMAND_CLASS_FIRMWARE_UPDATE_MD_V2 -->
+        <Class><id>0x80</id></Class>    <!-- COMMAND_CLASS_BATTERY -->
+        <Class><id>0x84</id></Class>    <!-- COMMAND_CLASS_WAKE_UP_V2 -->
+        <Class><id>0x85</id></Class>    <!-- COMMAND_CLASS_ASSOCIATION_V2-->
+        <Class><id>0x86</id></Class>    <!-- COMMAND_CLASS_VERSION_V2 -->
+        <Class><id>0x8F</id></Class>    <!-- COMMAND_CLASS_MULTI_CMD -->
+        <Class><id>0x98</id></Class>    <!-- COMMAND_CLASS_SECURITY -->
+    </CommandClasses>
+    <Configuration>
+        <Parameter>
+            <Index>1</Index>
+            <Type>byte</Type>
+            <Default>0</Default>
+            <Minimum>0</Minimum>
+            <Maximum>1</Maximum>
+            <Size>1</Size>
```

Comment:
Hi Chris,

 
I think I understood the manual wrong and 1 is not the right maximum value. 

This is the description of the parameter in the manual:

 
Name:

Basic Set OFF Level

 
Default:

0

 
Valid:

all

 
Description:

Setting the BASIC command value.

When the flood trigger off(0x00), send the BASIC CC to the group 2.

 
So from my new understanding one can decide here, which value between 0 and 255 will be send when the flood sensor triggers to off. The maximum value should be 255.

 
I will change the pr.

 
Regards and thank you for your support

Andreas

 
 
 
Von: Chris Jackson [mailto:notifications@github.com] 
Gesendet: Freitag, 8. Januar 2016 19:02
An: openhab/openhab <openhab@noreply.github.com>
Cc: rom69 <rom69@freenet.de>
Betreff: Re: [openhab] Add Philio PAT02-C Flood Sensor (#3732)

 
In bundles/binding/org.openhab.binding.zwave/database/philio/pat02c.xml:


> +        <Class><id>0x7A</id></Class>    <!-- COMMAND_CLASS_FIRMWARE_UPDATE_MD_V2 -->


> +        <Class><id>0x80</id></Class>    <!-- COMMAND_CLASS_BATTERY -->


> +        <Class><id>0x84</id></Class>    <!-- COMMAND_CLASS_WAKE_UP_V2 -->


> +        <Class><id>0x85</id></Class>    <!-- COMMAND_CLASS_ASSOCIATION_V2-->


> +        <Class><id>0x86</id></Class>    <!-- COMMAND_CLASS_VERSION_V2 -->


> +        <Class><id>0x8F</id></Class>    <!-- COMMAND_CLASS_MULTI_CMD -->


> +        <Class><id>0x98</id></Class>    <!-- COMMAND_CLASS_SECURITY -->


> +    </CommandClasses>


> +    <Configuration>


> +        <Parameter>


> +            <Index>1</Index>


> +            <Type>byte</Type>


> +            <Default>0</Default>


> +            <Minimum>0</Minimum>


> +            <Maximum>1</Maximum>


> +            <Size>1</Size>

I'm not sure I understand this parameter - it only has 2 options (0 and 1). Is this correct? Can we convert this to a list given there's only 2 options (does that make sense for this parameter?)

—
Reply to this email directly or view it on GitHub.